conpot/tests/test_http_server.py

```diff
@@ -216,7 +216,3 @@ def test_not_implemented_method(self):
             data=payload,
         )
         self.assertEqual(ret.status_code, 501)
-
```

Comment:
I remember adding these to run individual tests. Can remove then, but I don't see the point in doing so.

Reply:
In that case I would suggest https://docs.pytest.org/en/stable/usage.html#specifying-tests-selecting-tests